src/components/Tx/SendTx/CurrencySelect.jsx

```diff
@@ -0,0 +1,214 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import Identicon from '../../Identicon'
+
+export default class CurrencySelect extends Component {
+  static displayName = 'CurrencySelect'
+
+  static propTypes = {
+    etherWallet: PropTypes.object,
+    onClick: PropTypes.func,
+    tokens: PropTypes.array
+  }
+
+  static defaultProps = {
+    etherWallet: {
+      name: 'ETHER',
+      symbol: 'ETHER',
+      address: '0x123456BfA2A4DFAbdD1Fad5A1cb73a63345zzzzz',
```

Comment:
I don't think we should have a defaultProps of dummy values in the component, you handled that correctly in the stories

Reply:
I've updated - I think there's an argument for a default story and default values - namely, for when no props are passed through. I've removed the hard coded values but kept the keys as the code needs something iterable defined.